opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
> what is the use case where one needs a child context, but does not need a child span?

In fact that is the rationale I used for removing `TraceContext.newChild(SpanContext);`
 ref: https://github.com/opentracing/opentracing-java/pull/7#discussion_r50498361